File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// TemporaryEmailApi2 SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class TemporaryEmailApi2MakeContext
{
    public static function call(array $ctxmap, ?TemporaryEmailApi2Context $basectx): TemporaryEmailApi2Context
    {
        return new TemporaryEmailApi2Context($ctxmap, $basectx);
    }
}

File: .sdk/tm/rb/utility/make_context.rb
# TemporaryEmailApi2 SDK utility: make_context
require_relative '../core/context'
module TemporaryEmailApi2Utilities
  MakeContext = ->(ctxmap, basectx) {
    TemporaryEmailApi2Context.new(ctxmap, basectx)
  }
end
